### PR TITLE
React: Move signout button, add CCM logo

### LINKED ISSUE #363: Change signout button location and add CCM logo
Kind of open-ended for where the logo should go, but peopel said the signout button is more intuitive on the top right.

### PR DESCRIPTION
**Pull request by frewmack**
Closes #363

### CODE REVIEW

File: react/src/Navigation.tsx [hunk 1]
```diff
@@ -253,6 +258,11 @@ export default function Navigation({ signout, darkMode, toggleDarkMode }: Naviga
                         <Tooltip title={"Logged in as " + currentUser.username} arrow>
                             <AccountCircleIcon fontSize="large" />
                         </Tooltip>
+                        <Tooltip title="Log out">
```
Comment (kevinlul) [inline react/src/Navigation.tsx:261]:
```suggestion
                        <Tooltip title="Sign out">
```
Match how it was before (and sign in)

**Review by kevinlul — COMMENTED** [review]
I think we might need to change the app bar's colour if the logo is staying there because the contrast between the dark portion of the logo and the background isn't great.

image

**Comment by frewmack** [comment]
> I think we might need to change the app bar's colour if the logo is staying there because the contrast between the dark portion of the logo and the background isn't great.

Yeah I agree. I might look into how themes are handled in Material UI so that we can make changes like this more easily across the app.

**Comment by frewmack** [comment]
I decided to move the logo to the drawer, which gets hidden when the drawer is closed. The colours contrast nicely with both the light and dark theme backgrounds, and this avoids us having to come up with a colour scheme that accounts for the logo's colours for now.

**Review by kevinlul — APPROVED** [review]
light
dark

This is a W!

**Pull request merged**